src/SwedbankPay.Sdk.Infrastructure/PaymentOrders/PaymentOrder.cs

```diff
@@ -5,27 +5,27 @@
 
 namespace SwedbankPay.Sdk.PaymentOrders
 {
-    public class PaymentOrder
+    public class PaymentOrder : IPaymentOrder
```

Comment:
Codacy found an issue: [This class can't be instantiated; make its constructor 'public'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)